telemetry/definitions/commonDefinitions.json

```diff
@@ -1168,6 +1173,28 @@
             ],
             "passive": true
         },
+        {
+            "name": "toolkit_showNotification",
+            "description": "The toolkit tried to retrieve and show notification message",
+            "metadata": [
+                {
+                    "type": "result",
+                    "required": true
+                },
+                {
+                    "type": "notificationId",
```

Comment:
should we add a generic `id` type instead? or would it make sense to use the existing `name` or `source` here?

Reply:
`source` makes sense to me here. will update this

Reply:
`source` is more associated with "where did this thing take place?"

a generic `id` is a good suggestion, and would be appropriate here